src/components/jellyfinActions.ts

```diff
@@ -67,17 +67,16 @@ export function reportPlaybackStart(
 ): Promise<any> {
     clearBackdropInterval();
 
-    const url = JellyfinApi.createUrl('Sessions/Playing');
-
-    broadcastToMessageBus({
+    broadcastToMessageBus(
         //TODO: convert these to use a defined type in the type field
-        type: 'playbackstart',
-        data: getSenderReportingData($scope, reportingParams)
-    });
+        'playbackstart',
+        undefined,
+        getSenderReportingData($scope, reportingParams)
```

Comment:
Why this is no longer an options object. The undefined in the middle seems ugly.

Reply:
Undid that after I understood how interfaces work